ci/run-functional-tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+npx mocha --timeout 10m \
```

Comment:
I'll update to use `*` in the functional-test.yml eventually, but this is for tentative shell script during https://github.com/appium/appium/issues/21426